mindsdb/interfaces/model/model_controller.py

```diff
@@ -100,56 +105,6 @@ def _try_outdate_db_status(self, predictor_record):
         session.commit()
         return predictor_record
 
-
-    def create(self, name, company_id=None):
-        import mindsdb_native
-
-        original_name = name
-        name = f'{company_id}@@@@@{name}'
-
-        self._setup_for_creation(name, original_name, company_id=company_id)
-        predictor = mindsdb_native.Predictor(name=name, run_env={'trigger': 'mindsdb'})
-        return predictor
-    
-    def learn_for_update(self, name, from_data, to_predict, datasource_id, kwargs={}, company_id=None):
-        kwargs['join_learn_process'] = True
-        return self.learn(name, from_data, to_predict, datasource_id, kwargs, company_id, False)
-
-    def learn(self, name, from_data, to_predict, datasource_id, kwargs={}, company_id=None, save=True):
-        from mindsdb.interfaces.model.learn_process import LearnProcess, run_learn
-
-        create_process_mark('learn')
-        original_name = name
-        name = f'{company_id}@@@@@{name}'
-        join_learn_process = kwargs.get('join_learn_process', False)
-        
-        if save:
-            self._setup_for_creation(name, original_name, company_id=company_id)
-
-        if self.ray_based:
-            run_learn(
-                name=name,
-                db_name=original_name,
-                from_data=from_data,
-                to_predict=to_predict,
-                kwargs=kwargs,
-                datasource_id=datasource_id,
-                company_id=company_id,
-                save=save
-            )
-
-        else:
-            p = LearnProcess(name, original_name, from_data, to_predict, kwargs, datasource_id, company_id, save)
-            p.start()
-            if join_learn_process is True:
-                p.join()
-                if p.exitcode != 0:
-                    delete_process_mark('learn')
-                    raise Exception('Learning process failed !')
-
-        delete_process_mark('learn')
-        return 0
-
     def predict(self, name, pred_format, when_data=None, kwargs={}, company_id=None):
```

Comment:
For predict we might need a "backwards compatible" flag for now, that converts the new lightwood format into the old format that mindsdb was using.

Basically:

`prediction` -> `target_name`
`confidence` -> `target_name_confidence`
`lower_bound` -> `target_name_lower_bound`

Double check the actual names returned by native and by lightwood, and do the conversion here, and have a `backwards_compatible` argument (set to `True` by default)